pkg/dockerfile/generator_test.go

```diff
@@ -92,7 +92,7 @@ predict: predict.py:Predictor
 	require.NoError(t, err)
 
 	expected := `# syntax = docker/dockerfile:1.2
-FROM nvidia/cuda:11.0-cudnn8-devel-ubuntu16.04
+FROM nvidia/cuda:11.2.0-cudnn8-devel-ubuntu20.04
```

Comment:
I would expect the default Ubuntu and CUDA version to remain stable. What's going on here? Is it non-deterministically picking the Ubuntu version somehow?

Reply:
If you set `gpu: true` without specifying `cuda:` or including any python packages that pick a CUDA version for you, it defaults to the latest CUDA version. NVIDIA's base image uses Ubuntu 20.04 for CUDA 11.2.

Reply:
Gotchya. I have created an issue here: https://github.com/replicate/cog/issues/330